miio/integrations/vacuum/roborock/vacuum.py

```diff
@@ -131,6 +133,8 @@ def __init__(
     ):
         super().__init__(ip, token, start_id, debug, model=model)
         self.manual_seqnum = -1
+        self._multi_maps: Optional[MultiMapList] = None
+        self._multi_map_enum = None
```

Comment:
Do we really need to keep so much state in the class?

Reply:
Well we need to cache it somewhere, in normal use cases the amount of floors of a house does not change and therefore the multi_map information does not change.
I find it a bit resource wastefull to do IO on each status call to retrieve the same map information over and over again while we kind of know it will never change.

Reply:
If someone actually renames a floor or builds a new floor to there house, they just need to restart homeassistant or reload the integration.

Reply:
Okay, fair enough. I'm not that happy about having hidden state information stored in the device instance, but let's keep that for now.